test/test_person.py

```diff
@@ -0,0 +1,22 @@
+# test for create_openminds_person function in the main
+import pytest
+from bids2openminds.main import create_openminds_person
+import openminds.latest.core as omcore
+
+# Test data: (full_name, given_name, family_name)
+example_names = [("John Ronald Reuel Tolkien", "John Ronald Reuel", "Tolkien"),
```

Comment:
shouldn't there be an expected pass and an expected fail behavior tested?

Reply:
I integrated some negative behavior too, please verify it again.